### PR TITLE
Feature/map widget

### PR DESCRIPTION
**Pull request by ysdevys**
I suggest a merge at this stage because of the new dependencies and to avoid further complications with a later merge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,19 @@ You have to install the [Community Edition of Docker](https://store.docker.com/s
 
 Troubleshooting on Windows: If Virtualization is enabled, but wont work, try to [disable & enable Hyper-V](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v).
 
+### MapView widget
```
Comment (demirdagemir) [inline README.md]: I think you could either delete this or adjust it accordingly: The readme file we use it to explain functionalities for people -outside- who could use the source code, not to report our current status to our teammates.

File: index.html [hunk 2]
```diff
@@ -5,6 +5,7 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="http://cdn.leafletjs.com/leaflet/v0.7.7/leaflet.css" />
```
Comment (demirdagemir) [inline index.html]: We already have this in the dependencies. Please move this to the related component. (MapView.js)
You don't have to use a CDN, you could just do, inside MapView, `import 'leaflet/dist/leaflet.css'`

**Review by demirdagemir — CHANGES_REQUESTED** [review]
Please move the image inside `/style/img`, that's where we keep the images. You could then update the image link where you use it. The location pin image doesn't show up on the map component. (Image not found)

File: package.json [hunk 3]
```diff
@@ -14,9 +14,11 @@
   },
   "dependencies": {
     "html-webpack-plugin": "^2.28.0",
+    "leaflet": "^1.0.3",
```
Comment (demirdagemir) [inline package.json:17]: I'm not sure about this, I would like to ask the tutors, when should we use dependencies or devDependencies? I always get confused: I'm guessing webpack will bundle only dependencies inside a production environment, but we generally install only what we need. Don't we need every dependency? Except maybe webpack-dev-server. @kordianbruck @sacdallago

**Review by demirdagemir — CHANGES_REQUESTED** [review]

File: style/mapview.scss [hunk 4]
```diff
@@ -0,0 +1,5 @@
+.leaflet-container {
+    height: 400px;
+    width: 100%;
+	margin: 20px;
```
Comment (demirdagemir) [inline style/mapview.scss]: Could you also fix the indentation here?

File: package.json (hunk 3 above)
Comment (kordianbruck) [reply package.json:17]: Let me quote this:

> dependencies are required to run, devDependencies only to develop, e.g.: unit tests, Coffeescript to Javascript transpilation, minification, ...

https://stackoverflow.com/questions/18875674/whats-the-difference-between-dependencies-devdependencies-and-peerdependencies

**Comment by kordianbruck** [comment]
Fixes issue #42

File: style/mapview.scss (hunk 4 above)
Comment (ysdevys) [reply style/mapview.scss]: done

File: index.html (hunk 2 above)
Comment (ysdevys) [reply index.html]: thanks

File: README.md (hunk 1 above)
Comment (ysdevys) [reply README.md]: done

**Review by demirdagemir — APPROVED** [review]

**Comment by demirdagemir** [comment]
@ysdevys Can you please merge this?

**Pull request merged**